src/Compilers/Core/Portable/DiagnosticAnalyzer/ShadowCopyAnalyzerAssemblyLoader.cs

```diff
@@ -129,64 +138,65 @@ private void DeleteLeftoverDirectories()
             }
         }
 
-        protected override string PreparePathToLoad(string originalAnalyzerPath)
-        {
-            var mvid = AssemblyUtilities.ReadMvid(originalAnalyzerPath);
-
-            return PrepareLoad(_mvidPathMap, mvid, copyAnalyzerContents);
+        public bool IsAnalyzerPathHandled(string analyzerFilePath) => true;
```

Comment:
interestingly simple impl that i was not expecting.  wouldn't this then clash with the ProgramFilesResolver which said it handled some paths itself?

Reply:
(note: perhaps this is where ordering comes in.  don't have a good sense of what happens when multiple resolvers work this way).

Reply:
Added some more explicit docs about ordering to this interface definition.